themes/timeline/README.md

```diff
@@ -0,0 +1,25 @@
+# hugo-theme-timeline
```

Comment:
I see there are a lot of things that come with the theme, many css, fonts, language files. Can we keep the current homepage https://kochifoss.org as it is and only extract just the timeline?

i.e a new theme that keeps the current styling and just extracted timeline of the timeline theme